packages/gatsby-theme-styleguide/src/gatsby-components/Menu/MenuHeader.js

```diff
@@ -28,12 +34,18 @@ const getCurrentRelease = githubReleases => {
 }
 
 const MenuHeader = ({ siteTitle, githubReleases }) => (
-  <Header>
+  <React.Fragment>
```

Comment:
tips : instead of `<React.Fragment>... </React.Fragment>` you can just do `<>...</>`

Reply:
fixed